### PR TITLE
Update setting-up-playbooks.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added content for import, export, duplicate.

### CODE REVIEW

File: source/playbooks/setting-up-playbooks.rst [hunk 1]
```diff
@@ -111,3 +111,14 @@ You can change a playbook's configuration at any time, but changes will only be
  * To access the playbook dashboard, select the hyperlinked playbook name.
 
 4. Configure the playbook the same way a playbook is created or edited.
+
+Copying and reusing playbooks
+-----------------------------
+
+Playbooks are repeatable workflows and sometimes it's easier to copy and improve rather than start from scratch. You can do this by duplicating a playbook in the **Playbooks** screen. Select the **...** under **Actions** and then select **Duplicate**. The copied playbook will have **Copy of** appended to its original name which you can edit.
+
+You can also import playbooks between teams. From the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file.
```
Comment (justinegeffen) [inline source/playbooks/setting-up-playbooks.rst:120]: @crspeller, could you help with any missing content/context here. Thank you!

**Comment by justinegeffen** [comment]
@crspeller, if you have a moment to review this it'd be awesome. :)

**Review by cwarnermm — APPROVED** [review]

**Review by crspeller — CHANGES_REQUESTED** [review]

File: source/playbooks/setting-up-playbooks.rst [hunk 2]
```diff
@@ -111,3 +111,12 @@ You can change a playbook's configuration at any time, but changes will only be
...
+
+Lastly, if you want to create a backup of your playbook in JSON format, you can export it. To do this from the Playbooks screen, select the **...** under **Actions** and then select **Export**. Save the file to an appropriate location. The exported file can be used with your source control tool, to manage changes made to the playbook and allow you to rollback to previous versions.
```
[the remark this answers is not in the document]
Comment (justinegeffen) [reply source/playbooks/setting-up-playbooks.rst:122]: Ah interesting. Tao mentioned a use case around that but I may very well have misunderstood. :)

**Comment by justinegeffen** [comment]
Thanks, @crspeller! I've revised it to be simpler. The release announcement provides a good use case.

File: source/playbooks/setting-up-playbooks.rst [hunk 3]
```diff
@@ -111,3 +111,10 @@ You can change a playbook's configuration at any time, but changes will only be
  * To access the playbook dashboard, select the hyperlinked playbook name.
 
 4. Configure the playbook the same way a playbook is created or edited.
+
+Copying and reusing playbooks
+-----------------------------
+
+Playbooks are repeatable workflows and sometimes it's easier to copy and improve rather than start from scratch. You can do this by duplicating a playbook in the **Playbooks** screen. Select the **...** under **Actions** and then select **Duplicate**. The copied playbook will have **Copy of** appended to its original name which you can edit.
+
+You can import playbooks between teams. From the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file. You can also export any playbook to JSON to easily share with other Mattermost servers.
```
Comment (crspeller) [inline source/playbooks/setting-up-playbooks.rst:120]: Not sure what `between teams` means here. You can import playbooks.

**Review by crspeller — CHANGES_REQUESTED** [review]

File: source/playbooks/setting-up-playbooks.rst (hunk 3 above)
Comment (justinegeffen) [inline source/playbooks/setting-up-playbooks.rst:120]:
```suggestion
To import a playbook, go to the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file. You can also export any playbook to JSON to easily share with other Mattermost servers.
```

**Review by crspeller — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks for the super-quick review, @crspeller! :)

**Pull request merged**